src/cssLanguageService.ts

```diff
@@ -89,15 +97,39 @@ function handleCustomData(options?: LanguageServiceOptions) {
 
 export function getCSSLanguageService(options?: LanguageServiceOptions): LanguageService {
 	handleCustomData(options);
-	return createFacade(new Parser(), new CSSCompletion(), new CSSHover(), new CSSNavigation(), new CSSCodeActions(), new CSSValidation());
+	return createFacade(
+		new Parser(),
+		new CSSCompletion(),
+		new CSSHover(),
+		new CSSNavigation(),
```

Comment:
I would pass in the fileSystemProvider in the constructor
- no need to add `fsProvider` to the facade